.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: development
+    labels:
+      - "dependabot"
+    commit-message:
+      prefix: "[DEPENDABOT]"
```

Comment:
As we set up Dependabot using GitHub and this config file is not correct. I suggest removing this file again.